roles/grafana/templates/providers.yml.j2

```diff
@@ -0,0 +1,12 @@
+apiVersion: 1
+
+providers:
+  - name: "Dashboard provider"
+    orgId: 1
+    type: file
+    disableDeletion: false
+    updateIntervalSeconds: 10
+    allowUiUpdates: false
+    options:
+      path: /var/lib/grafana/dashboards
```

Comment:
Should this variable use the `grafana_dashboard_directory` variable?

Reply:
Added Variable in providers.yml.j2